### PR TITLE
New version: qBittorrent.qBittorrent version 4.6.4

### PR DESCRIPTION
**Pull request by SpecterShell**
Created by [🥟 Dumplings](https://github.com/SpecterShell/Dumplings) in Run [#3911](https://github.com/SpecterShell/Dumplings/actions/runs/8413731017).
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/146307)

### CODE REVIEW

File: manifests/q/qBittorrent/qBittorrent/4.6.4/qBittorrent.qBittorrent.locale.zh-CN.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+Tags:
+- p2p
+- 上传
+- 下载
+- 共享
+- 分享
+- 磁力链接
+- 磁链
+- 种子
+# ReleaseNotes:
```
Comment (drshtmstry) [inline manifests/q/qBittorrent/qBittorrent/4.6.4/qBittorrent.qBittorrent.locale.zh-CN.yaml:34]: v4.6.4 changelog:
- BUGFIX: Correctly adjust "Add New torrent" dialog position in all the cases (glassez)
- BUGFIX: Change "metadata received" stop condition behavior (glassez)
- BUGFIX: Add a small delay before processing the key input of search boxes (Chocobo1)
- BUGFIX: Ensure the profile path is pointing to a directory (Chocobo1)
- RSS: Use better icons for RSS articles (glassez)
- WINDOWS: NSIS: Update French, Hungarian translations (MarcDrieu, foxi69)
- LINUX: Fix sorting when ICU isn't used (Chocobo1)
- LINUX: Fix invisible tray icon on Plasma 6 (tehcneko)

File: manifests/q/qBittorrent/qBittorrent/4.6.4/qBittorrent.qBittorrent.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+- bittorrent
+- bt
+- download
+- magnet
+- p2p
+- share
+- sharing
+- torrent
+- upload
+# ReleaseNotes:
```
Comment (drshtmstry) [inline manifests/q/qBittorrent/qBittorrent/4.6.4/qBittorrent.qBittorrent.locale.en-US.yaml:35]: v4.6.4 changelog:
- BUGFIX: Correctly adjust "Add New torrent" dialog position in all the cases (glassez)
- BUGFIX: Change "metadata received" stop condition behavior (glassez)
- BUGFIX: Add a small delay before processing the key input of search boxes (Chocobo1)
- BUGFIX: Ensure the profile path is pointing to a directory (Chocobo1)
- RSS: Use better icons for RSS articles (glassez)
- WINDOWS: NSIS: Update French, Hungarian translations (MarcDrieu, foxi69)
- LINUX: Fix sorting when ICU isn't used (Chocobo1)
- LINUX: Fix invisible tray icon on Plasma 6 (tehcneko)

**Comment by drshtmstry** [comment]
- Resolves #146316

File: manifests/q/qBittorrent/qBittorrent/4.6.4/qBittorrent.qBittorrent.locale.en-US.yaml (hunk 2 above)
Comment (SpecterShell) [reply manifests/q/qBittorrent/qBittorrent/4.6.4/qBittorrent.qBittorrent.locale.en-US.yaml:35]: Nah I'm waiting for the one on https://www.qbittorrent.org/news

Comment (drshtmstry) [reply manifests/q/qBittorrent/qBittorrent/4.6.4/qBittorrent.qBittorrent.locale.en-US.yaml:35]: image

**Review by stephengillie — APPROVED** [review]

**Pull request merged**